clients/da-vinci-client/src/test/java/com/linkedin/davinci/ingestion/IsolatedIngestionBackendTest.java

```diff
@@ -0,0 +1,65 @@
+package com.linkedin.davinci.ingestion;
+
+import static com.linkedin.davinci.ingestion.main.MainPartitionIngestionStatus.ISOLATED;
+import static com.linkedin.davinci.ingestion.main.MainPartitionIngestionStatus.MAIN;
+import static com.linkedin.davinci.ingestion.main.MainPartitionIngestionStatus.NOT_EXIST;
+import static com.linkedin.venice.ingestion.protocol.enums.IngestionCommandType.START_CONSUMPTION;
+import static com.linkedin.venice.ingestion.protocol.enums.IngestionCommandType.STOP_CONSUMPTION;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doCallRealMethod;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.linkedin.davinci.ingestion.main.MainIngestionMonitorService;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Supplier;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class IsolatedIngestionBackendTest {
+  @Test
+  public void testBackendCanDirectCommandCorrectly() {
```

Comment:
Did you copy this test on the main branch and confirm that it failed?

Reply:
yes it is.